static/js/zamboni/files.js

```diff
@@ -1032,14 +1032,19 @@ $(document).ready(function() {
                 });
             } else if (json) {
                 var errors = false;
+
                 $.each(json.msg, function(k) {
                     if (json.msg[k] !== null) {
                         errors = true;
-                        $('<p>').text(json.msg[k]).appendTo($('#file-viewer div.error'));
+
+                        $('#validating').after($('<div>', {
+                            'class': 'notification-box error',
+                            'text': json.msg[k]
+                        }));
```

Comment:
Small note on this: The box isn't being shown / created if there are errors so we have to create the notification-box ourselves.